### PR TITLE
Update README.md http:// to https:// link.

### PR DESCRIPTION
**Pull request by davide93**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 Vegas – Backgrounds and Slideshows
 ==================================
 
-[![NPM version](https://badge.fury.io/js/vegas.svg)](http://badge.fury.io/js/vegas)
-[![Bower version](https://badge.fury.io/bo/vegas.svg)](http://badge.fury.io/bo/vegas)
+[![NPM version](httpss://badge.fury.io/js/vegas.svg)](https://badge.fury.io/js/vegas)
```
Comment (jaysalvat) [inline README.md]: Please fix ``httpss``.

Comment (jaysalvat) [inline README.md]: Please fix ``httpss``.

Comment (jaysalvat) [inline README.md]: Please fix ``httpss``.

Comment (jaysalvat) [inline README.md]: Please fix ``httpss``.

**Comment by davide93** [comment]
Fixed

**Comment by jaysalvat** [comment]
Merged. Thanks a lot davide93 !

**Pull request merged**